src/Hintman/Suggestion/TrailingSpaces.hs

```diff
@@ -0,0 +1,13 @@
+module Hintman.Suggestion.TrailingSpaces
+       ( suggest
+       ) where
+
+import Data.Text (stripEnd)
+import Hintman.Suggestion.Core (Line (..), Suggest (..))
+
+
+suggest :: FilePath -> [Line] -> [Suggest]
+suggest path =
+    foldr (\Line{..} acc ->
```

Comment:
You can use [`mapMaybe`](https://hackage.haskell.org/package/relude-0.3.0/docs/Relude-Monad-Reexport.html#v:mapMaybe) function instead if you will add `Line -> Maybe Suggestion` function.
```haskell
suggest path = mapMaybe foo 
  where
    foo :: Line -> Maybe Suggestion
    foo = ...
```